Thanks @jimwitte for contributing to the Ansible documentation. Your PR currently has two small flaws standing between it and a clean CI run:

```
lib/ansible/modules/crypto/get_certificate.py:110:1: W293 blank line contains whitespace
lib/ansible/modules/crypto/get_certificate.py:115:161: E501 line too long (165 > 160 characters)
```

Can you fix those two issues?

--- a/lib/ansible/modules/crypto/get_certificate.py
+++ b/lib/ansible/modules/crypto/get_certificate.py
@@ -107,12 +107,12 @@ EXAMPLES = '''
   delegate_to: localhost
   run_once: true
   register: cert
-  
+
 - name: How many days until cert expires
   debug:
     msg: "cert expires in: {{ expire_days }} days."
   vars:
-    expire_days: "{{ (( get_certificate_result.not_after | to_datetime('%Y%m%d%H%M%SZ')) - (ansible_date_time.iso8601 | to_datetime('%Y-%m-%dT%H:%M:%SZ')) ).days }}"
+    expire_days: "{{ (( cert.not_after | to_datetime('%Y%m%d%H%M%SZ')) - (ansible_date_time.iso8601 | to_datetime('%Y-%m-%dT%H:%M:%SZ')) ).days }}"
 '''
 
 import traceback

--- a/lib/ansible/modules/crypto/get_certificate.py
+++ b/lib/ansible/modules/crypto/get_certificate.py
@@ -108,7 +108,7 @@ EXAMPLES = '''
   run_once: true
   register: cert
   
-- name: how many days until cert expires
+- name: How many days until cert expires
   debug:
     msg: "cert expires in: {{ expire_days }} days."
   vars:

